app/controllers/katello/api/v2/repositories_bulk_actions_controller.rb

```diff
@@ -18,40 +18,11 @@ class Api::V2::RepositoriesBulkActionsController < Api::V2::ApiController
     param :ids, Array, :desc => N_("List of repository ids"), :required => true
     def destroy_repositories
       deletion_authorized_repositories = @repositories.deletable
-
       unpromoted_repos = deletion_authorized_repositories.reject { |repo| repo.promoted? }
-
       deletable_repositories = unpromoted_repos.reject { |repo| repo.redhat? }
 
-      deletable_repositories.each do |repository|
-        trigger(::Actions::Katello::Repository::Destroy, repository)
-      end
-
-      messages1 = format_bulk_action_messages(
-        :success    => "",
-        :error      => _("You do not have permissions to delete %s"),
-        :models     => @repositories,
-        :authorized => deletion_authorized_repositories
-      )
-
-      messages2 = format_bulk_action_messages(
-        :success    => "",
-        :error      => _("Repository %s cannot be deleted since it has already been included in a published Content View."),
-        :models     => deletion_authorized_repositories,
-        :authorized => unpromoted_repos
-      )
-
-      messages3 = format_bulk_action_messages(
-        :success    => _("Successfully initiated deletion for %s repositories, you are free to leave this page."),
-        :error      => _("Repository %s cannot be deleted since they are Red Hat repositories."),
-        :models     => unpromoted_repos,
-        :authorized => deletable_repositories
-      )
-
-      messages3[:error] = messages3[:error] + messages1[:error] + messages2[:error]
-
-      respond_for_show :template => 'bulk_action', :resource_name => 'common',
```

Comment:
Ah, good, point, I can bring those back and display as errors on submitting the bulk action.